examples/autoqasm/1_Getting_started_with_AutoQASM.ipynb

```diff
@@ -80,8 +80,7 @@
     }
```

Comment:
Looks good. Please update the text, too. For example: "The quantum program is returned by calling the decorated bell_state function."